datadog_checks_dev/datadog_checks/dev/tooling/manifest_validator/common/validator.py

```diff
@@ -317,3 +313,40 @@ class VersionValidator(BaseManifestValidator):
     def validate(self, check_name, decoded, fix):
         if decoded.get('manifest_version', V2_STRING) == V1_STRING:
             self.fail('Manifest version must be >= 2.0.0')
+
+
+class ModifiedAttributesValidator(BaseManifestValidator):
```

Comment:
I think this a bad name. Do you have suggestions?

Reply:
How about merging this with the existing `ImmutableAttributesValidator`? The logic is almost identical, it would be a matter of adding a way to mark some fields as warning-only.